test/integration/dialects/postgres/data-types.test.js

```diff
@@ -6,6 +6,8 @@ const Support = require('../../support');
 const dialect = Support.getTestDialect();
```

Comment:
Tests in this file should be removed after this `shortCrs` option is used for MySQL/MariaDB. https://github.com/sequelize/sequelize/blob/master/test/integration/model/geometry.test.js should be modified after everything is using `shortCrs`

Reply:
@sushantdhiman @papb Hello,

Done all requested changes, tests are passing except irrelevant TIME tests for SQLITE.